infra/base-images/base-builder/Dockerfile

```diff
@@ -105,6 +106,7 @@ COPY compile compile_afl compile_dataflow compile_libfuzzer compile_honggfuzz \
     precompile_honggfuzz srcmap write_labels.py /usr/local/bin/
 
 COPY detect_repo.py /opt/cifuzz/
+COPY ossfuzz_test.go /root/go/
```

Comment:
do we need this scrip tin the base-builder image?

Reply:
We need this somewhere.
It is meant to be generic for all projects. (but it can be in each project)
So, it is related to `compile_fuzzer` being generic or not

What do you want for now ? specific by projet or generic ?

Reply:
Leave it like this for now, I need to think more about how we can unify the whole compile_fuzzer thing

Reply:
Let's rename it to `ossfuzz_coverage_runner.go`